tailormap-components/projects/core/src/lib/user-interface/attributelist/attributelist-panel/attributelist-panel.component.ts

```diff
@@ -0,0 +1,143 @@
+/**============================================================================
+ *===========================================================================*/
+
+import { Component, ElementRef, OnInit, AfterViewInit, ViewChild,
+         Renderer2, HostBinding} from '@angular/core';
+
+import {ILayer} from '../layer.model';
+import {LayerService} from '../layer.service';
+import {LayoutService, ILayoutComponent, Dock, WindowState} from '../../panel-resizer/layout.service';
+
+import {PanelResizerComponent} from '../../panel-resizer/panel-resizer.component';
+
+import {MatToolbar} from '@angular/material/toolbar';
+
+@Component({
+  selector: 'tailormap-attributelist-panel',
+  templateUrl: './attributelist-panel.component.html',
+  styleUrls: ['./attributelist-panel.component.css']
+})
+export class AttributelistPanelComponent implements OnInit, AfterViewInit, ILayoutComponent {
+
+  // For the layout service.
+  @ViewChild('captionbar') captionbar: MatToolbar;
+  @ViewChild("panelResizer") panelResizer: PanelResizerComponent;
+
+  // For the layout service.
+  // TODO: DIT IN EEN APARTE STRUCTURE/CLASS???
+  canResize = true;
+  componentRef = this.elementRef;
```

Comment:
Deze wordt in de constructor al geset, dus volgens mij is dit een self-assignment

Reply:
"componentRef" was onderdeel van de interface. Zit nu in de layoutConfig. Dus opgelost.